src/main/resources/standardTestCases/globbingindex/globbingindex.wdl

```diff
@@ -0,0 +1,60 @@
+task glob_task {
+    command {
+        echo "staticFile" > staticFile.txt
+        echo "staticArray" > staticArray.txt
+        echo "globFile" > globFile.txt
+        echo "globArray" > globArray.txt
+    }
+
+    output {
+        Array[File] staticFiles = ["staticFile.txt"]
+        Array[File] staticRef = staticFiles
+        File staticFile = staticFiles[0]
+        Array[File] staticArray = ["staticArray.txt"]
+        File globFile = glob("globFile.txt")[0]
+        Array[File] globArray = glob("globArray.txt")
+    }
+}
+
+task read_task {
+    Array[File] staticFiles
+    Array[File] staticRef
+    File staticFile
+    Array[File] staticArray
+    File globFile
+    Array[File] globArray
+
+    command { : }
```

Comment:
http://stackoverflow.com/questions/3224878/what-is-the-purpose-of-the-colon-gnu-bash-builtin/3224910#3224910